pkg/R/sparkRClient.R

```diff
@@ -2,7 +2,7 @@
 
 # Creates a SparkR client connection object
 # if one doesn't already exist
-connectBackend <- function(hostname, port, timeout = 6000) {
+connectBackend <- function(hostname, port, timeout = 6) {
```

Comment:
Good point, I will change it back.

Should we use blocking mode? It's possible that a job may take a few hours. In PySpark, it's in blocking mode.